Javascript/Housey pricey/house.js

```diff
@@ -0,0 +1,37 @@
+// Peter's parameters
+let peterWidth = 8;
```

Comment:
Imagine, you don't have context of what the assignment is about and you read those variable names. They might not be the most clear.
`peterHouseWidth` would be more appropriate and descriptive. Also you can set them as `const`.

Reply:
****Changed.

Reply:
I am glad you are getting things changed. It's a great way to go through the code and do even more learning.
It's not necessary to reply to every comment. Once you push a commit, I get an email notification. If you write a comment eg "Changed", "Fixed" or something else, I get a new email, but each time a new commit is pushed that makes changes to a particular line of code that I've commented on, I can see this "Outdated" label, which means that my comment has been addressed ;) This is just a heads-up :)
![Screenshot 2024-06-05 at 5 20 04 PM](https://github.com/l0rians/hyf-homework/assets/52400967/f9ba8905-70a2-42ab-b4f6-34cb1945b1a9)